virttest/qemu_vm.py

```diff
@@ -204,6 +203,10 @@ def __init__(self, name, params, root_dir, address_cache, state=None):
         self.last_boot_index = 0
         self.last_driver_index = 0
 
+    @property
+    def vcpu_threads(self):
+        return self.get_vcpu_pids(debug=True)
```

Comment:
```suggestion
        return self._get_vcpu_pids(debug=True)
```
I think that we should set `get_vcpu_pids` to protect the method if wanna use `@property` since one of both `vcpu_threads` and `get_vcpu_pids` is redundant. 
Calling `vm.vcpu_threads` is equal to `vm.get_vcpu_pids`

Reply:
Yes, i see this problem, but both `vm.vcpu_threads` and `vm.get_vcpu_pids` are used in tp-qemu now, so i did not change it.
To use protect `get_vcpu_pids`, we need update the other calling in tp-qemu(tp-libvirt and tp-spice do not use it).  If needed, i can update them.

Reply:
Okay, thanks.

Reply:
Keep `get_vcpu_pids`, support use `debug=True` in some situation, Thanks!